### ISSUE TITLE
Subclavian artery stenosis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10265]
**Preferred term label:**
Subclavian artery stenosis

**Synonyms**

**Definition (free text, please give PubMed ID)**
There seems to be a UMLS code for this but Login.gov is not working at this time and I can't get the UMLS code.  It is not in MedGen.

**Parent term (use hpo.jax.org/app)**
Possibly https://hpo.jax.org/app/browse/term/HP:0031251

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Subclavian steal syndrome (SSS)

**Your nano-attribution (ORCID)**

**Comment by MickeySegal** [comment]
Login.gov is back up and the relevant CUIs are 4303724, 4706772 and 4047831, which are also in MedGen.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000038
- label: Subclavian artery stenosis
- parents: [Abnormal subclavian artery morphology (HP:0031251)](https://hpo.jax.org/app/browse/term/HP:0031251)
- pmid: PMID:21078767
- def: Abnormal narrowing of the subclavian artery.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10265
- synonyms:
